Apply composer update

nanasess opened the pull request:
<!-- 以下を参考にコメントを作成してください。 -->

## 概要(Overview・Refs Issue)
<!-- PullRequestの目的、関連するIssue番号など -->
- composer update を実行
  - composer2.1.x へのバージョンアップに伴い、https://github.com/EC-CUBE/eccube-plugin-installer/pull/19 の対応が必要
- PHPStan のエラー修正

<details>

  - Removing php-coveralls/php-coveralls (v2.4.3)
  - Upgrading codeception/codeception (4.1.20 => 4.1.21)
  - Upgrading codeception/module-webdriver (1.2.0 => 1.2.1)
  - Upgrading composer/ca-bundle (1.2.9 => 1.2.10)
  - Upgrading composer/composer (2.0.12 => 2.1.3)
  - Locking composer/metadata-minifier (1.0.0)
  - Upgrading composer/package-versions-deprecated (1.11.99.1 => 1.11.99.2)
  - Upgrading composer/semver (3.2.4 => 3.2.5)
  - Upgrading composer/xdebug-handler (1.4.6 => 2.0.1)
  - Upgrading doctrine/annotations (1.12.1 => 1.13.1)
  - Upgrading doctrine/cache (1.10.2 => 1.12.0)
  - Upgrading doctrine/dbal (2.13.0 => 2.13.2)
  - Upgrading doctrine/inflector (1.3.1 => 1.4.4)
  - Upgrading friendsofphp/php-cs-fixer (v2.18.4 => v2.19.0)
  - Upgrading friendsofphp/proxy-manager-lts (v1.0.3 => v1.0.5)
  - Upgrading guzzlehttp/psr7 (1.8.1 => 1.8.2)
  - Upgrading monolog/monolog (1.26.0 => 1.26.1)
  - Upgrading nikic/php-parser (v4.10.4 => v4.11.0)
  - Upgrading php-webdriver/webdriver (1.10.0 => 1.11.1)
  - Upgrading phpstan/phpstan (0.12.91 => 0.12.92)
  - Upgrading psr/log (1.1.3 => 1.1.4)
  - Upgrading skorp/detect-incompatible-samesite-useragents (1.0.0 => 1.0.1)
  - Upgrading symfony/asset (v4.4.20 => v4.4.25)
  - Upgrading symfony/browser-kit (v4.4.20 => v4.4.25)
  - Upgrading symfony/cache (v4.4.21 => v4.4.26)
  - Upgrading symfony/config (v4.4.20 => v4.4.26)
  - Upgrading symfony/console (v4.4.21 => v4.4.26)
  - Upgrading symfony/css-selector (v4.4.20 => v4.4.25)
  - Upgrading symfony/debug (v4.4.20 => v4.4.25)
  - Upgrading symfony/dependency-injection (v4.4.21 => v4.4.26)
  - Upgrading symfony/deprecation-contracts (v2.2.0 => v2.4.0)
  - Upgrading symfony/doctrine-bridge (v4.4.21 => v4.4.25)
  - Upgrading symfony/dom-crawler (v4.4.20 => v4.4.25)
  - Upgrading symfony/dotenv (v4.4.20 => v4.4.25)
  - Upgrading symfony/error-handler (v4.4.21 => v4.4.26)
  - Upgrading symfony/event-dispatcher (v4.4.20 => v4.4.25)
  - Upgrading symfony/expression-language (v4.4.20 => v4.4.25)
  - Upgrading symfony/filesystem (v4.4.21 => v4.4.26)
  - Upgrading symfony/finder (v4.4.20 => v4.4.25)
  - Upgrading symfony/flex (v1.12.2 => v1.13.3)
  - Upgrading symfony/form (v4.4.21 => v4.4.26)
  - Upgrading symfony/framework-bundle (v4.4.21 => v4.4.26)
  - Upgrading symfony/http-foundation (v4.4.20 => v4.4.26)
  - Upgrading symfony/http-kernel (v4.4.21 => v4.4.26)
  - Upgrading symfony/inflector (v4.4.21 => v4.4.25)
  - Upgrading symfony/intl (v4.4.20 => v4.4.25)
  - Upgrading symfony/maker-bundle (v1.30.2 => v1.33.0)
  - Upgrading symfony/mime (v4.4.21 => v4.4.26)
  - Upgrading symfony/monolog-bridge (v4.4.21 => v4.4.26)
  - Upgrading symfony/options-resolver (v4.4.20 => v4.4.25)
  - Upgrading symfony/phpunit-bridge (v4.4.21 => v4.4.26)
  - Upgrading symfony/polyfill-ctype (v1.22.1 => v1.23.0)
  - Upgrading symfony/polyfill-iconv (v1.22.1 => v1.23.0)
  - Upgrading symfony/polyfill-intl-icu (v1.22.1 => v1.23.0)
  - Upgrading symfony/polyfill-intl-idn (v1.22.1 => v1.23.0)
  - Upgrading symfony/polyfill-intl-normalizer (v1.22.1 => v1.23.0)
  - Upgrading symfony/polyfill-mbstring (v1.22.1 => v1.23.0)
  - Upgrading symfony/polyfill-php72 (v1.22.1 => v1.23.0)
  - Upgrading symfony/polyfill-php73 (v1.22.1 => v1.23.0)
  - Upgrading symfony/polyfill-php80 (v1.22.1 => v1.23.0)
  - Locking symfony/polyfill-php81 (v1.23.0)
  - Upgrading symfony/process (v4.4.20 => v4.4.26)
  - Upgrading symfony/property-access (v4.4.20 => v4.4.25)
  - Upgrading symfony/proxy-manager-bridge (v4.4.20 => v4.4.25)
  - Upgrading symfony/routing (v4.4.20 => v4.4.25)
  - Upgrading symfony/security (v4.4.21 => v4.4.26)
  - Upgrading symfony/security-bundle (v4.4.21 => v4.4.26)
  - Upgrading symfony/serializer (v4.4.20 => v4.4.26)
  - Upgrading symfony/stopwatch (v4.4.20 => v4.4.25)
  - Upgrading symfony/templating (v4.4.20 => v4.4.25)
  - Upgrading symfony/translation (v4.4.21 => v4.4.26)
  - Upgrading symfony/twig-bridge (v4.4.21 => v4.4.26)
  - Upgrading symfony/twig-bundle (v4.4.20 => v4.4.26)
  - Upgrading symfony/validator (v4.4.21 => v4.4.26)
  - Upgrading symfony/var-dumper (v4.4.21 => v4.4.26)
  - Upgrading symfony/var-exporter (v4.4.20 => v4.4.26)
  - Upgrading symfony/web-profiler-bundle (v4.4.21 => v4.4.26)
  - Upgrading symfony/web-server-bundle (v4.4.21 => v4.4.26)
  - Upgrading symfony/workflow (v4.4.20 => v4.4.25)
  - Upgrading symfony/yaml (v4.4.21 => v4.4.26)

</details>

## 方針(Policy)
<!-- このPullRequestを作るにあたって考慮したものや除外した内容
  - 例）Symfony のXXにならって作成、3系で同様の仕様があったため など -->
- composer update を実行

## 実装に関する補足(Appendix)
<!-- コードだけではわかりづらい点など、実装するにあたってレビューアに追加で伝えておきたいこと -->
以下の対応後、再度 composer update を実行します
https://github.com/EC-CUBE/eccube-plugin-installer/pull/19

## テスト（Test)
<!-- テストを行っている範囲など、レビューアが安心できるような情報 -->
eccube-plugin-installer を修正した状態で GitHub Actions が通るのを確認

## 相談（Discussion）
<!-- 相談したいことや意見を求めたいこと -->
bin/console コマンドのテストケースも作りたい

## マイナーバージョン互換性保持のための制限事項チェックリスト
<!-- マイナーバージョンでは、機能・プラグイン・デザインテンプレート互換性を損なう変更は原則取り込みません。 -->

- [ ] 既存機能の仕様変更
- [ ] フックポイントの呼び出しタイミングの変更
- [ ] フックポイントのパラメータの削除・データ型の変更
- [ ] twigファイルに渡しているパラメータの削除・データ型の変更
- [ ] Serviceクラスの公開関数の、引数の削除・データ型の変更
- [ ] 入出力ファイル(CSVなど)のフォーマット変更

## レビュワー確認項目

- [ ] 動作確認
- [ ] コードレビュー
- [ ] E2E/Unit テスト確認(テストの追加・変更が必要かどうか)
- [ ] 互換性が保持されているか
- [ ] セキュリティ上の問題がないか

nanasess committed:
Apply composer update

  - Removing php-coveralls/php-coveralls (v2.4.3)
  - Upgrading codeception/codeception (4.1.20 => 4.1.21)
  - Upgrading codeception/module-webdriver (1.2.0 => 1.2.1)
  - Upgrading composer/ca-bundle (1.2.9 => 1.2.10)
  - Upgrading composer/composer (2.0.12 => 2.1.3)
  - Locking composer/metadata-minifier (1.0.0)
  - Upgrading composer/package-versions-deprecated (1.11.99.1 => 1.11.99.2)
  - Upgrading composer/semver (3.2.4 => 3.2.5)
  - Upgrading composer/xdebug-handler (1.4.6 => 2.0.1)
  - Upgrading doctrine/annotations (1.12.1 => 1.13.1)
  - Upgrading doctrine/cache (1.10.2 => 1.12.0)
  - Upgrading doctrine/dbal (2.13.0 => 2.13.2)
  - Upgrading doctrine/inflector (1.3.1 => 1.4.4)
  - Upgrading friendsofphp/php-cs-fixer (v2.18.4 => v2.19.0)
  - Upgrading friendsofphp/proxy-manager-lts (v1.0.3 => v1.0.5)
  - Upgrading guzzlehttp/psr7 (1.8.1 => 1.8.2)
  - Upgrading monolog/monolog (1.26.0 => 1.26.1)
  - Upgrading nikic/php-parser (v4.10.4 => v4.11.0)
  - Upgrading php-webdriver/webdriver (1.10.0 => 1.11.1)
  - Upgrading phpstan/phpstan (0.12.91 => 0.12.92)
  - Upgrading psr/log (1.1.3 => 1.1.4)
  - Upgrading skorp/detect-incompatible-samesite-useragents (1.0.0 => 1.0.1)
  - Upgrading symfony/asset (v4.4.20 => v4.4.25)
  - Upgrading symfony/browser-kit (v4.4.20 => v4.4.25)
  - Upgrading symfony/cache (v4.4.21 => v4.4.26)
  - Upgrading symfony/config (v4.4.20 => v4.4.26)
  - Upgrading symfony/console (v4.4.21 => v4.4.26)
  - Upgrading symfony/css-selector (v4.4.20 => v4.4.25)
  - Upgrading symfony/debug (v4.4.20 => v4.4.25)
  - Upgrading symfony/dependency-injection (v4.4.21 => v4.4.26)
  - Upgrading symfony/deprecation-contracts (v2.2.0 => v2.4.0)
  - Upgrading symfony/doctrine-bridge (v4.4.21 => v4.4.25)
  - Upgrading symfony/dom-crawler (v4.4.20 => v4.4.25)
  - Upgrading symfony/dotenv (v4.4.20 => v4.4.25)
  - Upgrading symfony/error-handler (v4.4.21 => v4.4.26)
  - Upgrading symfony/event-dispatcher (v4.4.20 => v4.4.25)
  - Upgrading symfony/expression-language (v4.4.20 => v4.4.25)
  - Upgrading symfony/filesystem (v4.4.21 => v4.4.26)
  - Upgrading symfony/finder (v4.4.20 => v4.4.25)
  - Upgrading symfony/flex (v1.12.2 => v1.13.3)
  - Upgrading symfony/form (v4.4.21 => v4.4.26)
  - Upgrading symfony/framework-bundle (v4.4.21 => v4.4.26)
  - Upgrading symfony/http-foundation (v4.4.20 => v4.4.26)
  - Upgrading symfony/http-kernel (v4.4.21 => v4.4.26)
  - Upgrading symfony/inflector (v4.4.21 => v4.4.25)
  - Upgrading symfony/intl (v4.4.20 => v4.4.25)
  - Upgrading symfony/maker-bundle (v1.30.2 => v1.33.0)
  - Upgrading symfony/mime (v4.4.21 => v4.4.26)
  - Upgrading symfony/monolog-bridge (v4.4.21 => v4.4.26)
  - Upgrading symfony/options-resolver (v4.4.20 => v4.4.25)
  - Upgrading symfony/phpunit-bridge (v4.4.21 => v4.4.26)
  - Upgrading symfony/polyfill-ctype (v1.22.1 => v1.23.0)
  - Upgrading symfony/polyfill-iconv (v1.22.1 => v1.23.0)
  - Upgrading symfony/polyfill-intl-icu (v1.22.1 => v1.23.0)
  - Upgrading symfony/polyfill-intl-idn (v1.22.1 => v1.23.0)
  - Upgrading symfony/polyfill-intl-normalizer (v1.22.1 => v1.23.0)
  - Upgrading symfony/polyfill-mbstring (v1.22.1 => v1.23.0)
  - Upgrading symfony/polyfill-php72 (v1.22.1 => v1.23.0)
  - Upgrading symfony/polyfill-php73 (v1.22.1 => v1.23.0)
  - Upgrading symfony/polyfill-php80 (v1.22.1 => v1.23.0)
  - Locking symfony/polyfill-php81 (v1.23.0)
  - Upgrading symfony/process (v4.4.20 => v4.4.26)
  - Upgrading symfony/property-access (v4.4.20 => v4.4.25)
  - Upgrading symfony/proxy-manager-bridge (v4.4.20 => v4.4.25)
  - Upgrading symfony/routing (v4.4.20 => v4.4.25)
  - Upgrading symfony/security (v4.4.21 => v4.4.26)
  - Upgrading symfony/security-bundle (v4.4.21 => v4.4.26)
  - Upgrading symfony/serializer (v4.4.20 => v4.4.26)
  - Upgrading symfony/stopwatch (v4.4.20 => v4.4.25)
  - Upgrading symfony/templating (v4.4.20 => v4.4.25)
  - Upgrading symfony/translation (v4.4.21 => v4.4.26)
  - Upgrading symfony/twig-bridge (v4.4.21 => v4.4.26)
  - Upgrading symfony/twig-bundle (v4.4.20 => v4.4.26)
  - Upgrading symfony/validator (v4.4.21 => v4.4.26)
  - Upgrading symfony/var-dumper (v4.4.21 => v4.4.26)
  - Upgrading symfony/var-exporter (v4.4.20 => v4.4.26)
  - Upgrading symfony/web-profiler-bundle (v4.4.21 => v4.4.26)
  - Upgrading symfony/web-server-bundle (v4.4.21 => v4.4.26)
  - Upgrading symfony/workflow (v4.4.20 => v4.4.25)
  - Upgrading symfony/yaml (v4.4.21 => v4.4.26)

nanasess committed:
Fix errors found by phpstan

okazy commented:
https://github.com/EC-CUBE/eccube-plugin-installer/pull/19 を取り込んでリリースタグを打ちました。

https://github.com/EC-CUBE/eccube-plugin-installer/releases/tag/2.0.1
https://packagist.org/packages/ec-cube/plugin-installer#2.0.1

okazy commented:
@nanasess 
再度 composer update の実行をお願いできますでしょうか？

nanasess committed:
Apply composer update

  - Upgrading doctrine/cache (1.12.0 => 1.12.1)
  - Upgrading ec-cube/plugin-installer (2.0.0 => 2.0.1)
  - Upgrading mikey179/vfsstream (v1.6.8 => v1.6.9)
  - Upgrading phpunit/php-file-iterator (2.0.3 => 2.0.4)

okazy reviewed (approved):
差分確認しました。
動作が変わりそうな変更はありませんでした。

okazy commented:
ありがとうございます、マージしました。
